stable/goldilocks/README.md

```diff
@@ -1,12 +1,17 @@
-# goldilocks
+<div align="center">
+<a href="https://github.com/FairwindsOps/goldilocks"><img src="/logo.svg" height="150" alt="Goldilocks" style="padding-bottom: 20px" /></a>
```

Comment:
I think you need to remove the `/` in `/logo.svg` to reference the logo in this directory. Can also probably remove the `icon.svg` file

Reply:
Fixed. Icon needs to be there for the Chart.yaml reference